rpm/generic/zfs.spec.in

```diff
@@ -314,7 +314,7 @@ This package provides a python wrapper for the libzfs_core C library.
 Summary:        Initramfs module
 Group:          System Environment/Kernel
 Requires:       %{name}%{?_isa} = %{version}-%{release}
-Requires:       %{name} = %{version}-%{release}
+Requires:       %{name}%{?_isa} = %{version}-%{release}
```

Comment:
These two lines seem to be the same.  Is that what we want?

Reply:
```diff
Requires:       %{name}%{?_isa} = %{version}-%{release}
+Requires:       %{name}%{?_isa} = %{version}-%{release}
```

Reply:
Oh, you're right, we have a redundant dependency here.